up-core-api/uprotocol/core/udiscovery/v3/udiscovery_replicator.proto

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ * 
+ * This program and the accompanying materials are made available under
+ * the terms of the Apache License Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileType: SOURCE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+syntax = "proto3";
+
+package uprotocol.core.udiscovery.v3;
+import "google/protobuf/timestamp.proto";
+import "uprotocol/uoptions.proto";
+import "uprotocol/v1/uri.proto";
+import "uprotocol/core/udiscovery/v3/udiscovery.proto";
+ 
+option java_package = "org.eclipse.uprotocol.core.udiscovery.v3";
+option java_outer_classname = "UDiscoveryReplicatorProto";
+option java_multiple_files = true;
+ 
+  
+// UDiscovery Replicator is a set of APIs that are used internal to UDiscovery to populate and replicate
+// information in the uDiscovery database. This API is for internal (to UDiscovery service) use only and 
+// not accessible for client uEs to use.
+ service UDiscoveryReplicator {
+  option (uprotocol.service_name) = "core.udiscovery"; // Service name
```

Comment:
FMPOV this should have a name that is different from the `UDiscovery` service defined in the other proto.